.github/fabricbot.json

```diff
@@ -1,3600 +0,0 @@
-                {
-                  "name": "addedToProjectColumn",
-                  "parameters": {
-                    "projectName": "Bugs 7.1",
-                    "columnName": ""
-                  }
-                },
-                {
-                  "name": "addedToProjectColumn",
-                  "parameters": {
-                    "projectName": "Features 7.1"
-                  }
-                },
-                {
-                  "name": "addedToProjectColumn",
-                  "parameters": {
-                    "projectName": "Technical 7.1"
-                  }
-                },
-                {
-                  "name": "addedToProjectColumn",
-                  "parameters": {
-                    "projectName": "Samples 7.1"
-                  }
-                }
-              ]
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "actions": [
-          {
-            "name": "addMilestone",
-            "parameters": {
-              "milestoneName": "7.1"
-            }
-          }
-        ],
-        "taskName": "Add milestone if the issue is added in the project column (7.1)"
-      }
-    },
-    {
-      "taskType": "trigger",
-      "capabilityId": "IssueResponder",
-      "subCapability": "IssuesOnlyResponder",
-      "version": "1.0",
-      "id": "fWjQeVQ87",
-      "config": {
-        "conditions": {
-          "operator": "and",
-          "operands": [
-            {
-              "name": "isOpen",
-              "parameters": {}
-            },
-            {
-              "operator": "or",
-              "operands": [
-                {
-                  "name": "removedFromProjectColumn",
-                  "parameters": {
-                    "projectName": "Samples 7.1"
-                  }
-                },
-                {
-                  "name": "removedFromProjectColumn",
-                  "parameters": {
-                    "projectName": "Features 7.1"
-                  }
-                },
-                {
-                  "name": "removedFromProjectColumn",
-                  "parameters": {
-                    "projectName": "Technical 7.1"
-                  }
-                },
-                {
-                  "name": "removedFromProjectColumn",
-                  "parameters": {
-                    "milestoneName": "Technical 7.1",
-                    "projectName": "Technical 7.1"
-                  }
-                }
-              ]
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "actions": [
-          {
-            "name": "removeMilestone",
-            "parameters": {}
-          }
-        ],
-        "taskName": "Remove milestone if the issue is removed from the project column"
```

Comment:
Project board automation is not supported in the new service, so related tasks were not ported.